geopandas/geodataframe.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, *args, **kwargs):
 
         # need to set this before calling self['geometry'], because
         # getitem accesses crs
-        self._crs = crs if crs is not None else None
+        self._crs = None if not crs else CRS.from_user_input(crs)
```

Comment:
@jorisvandenbossche this is very minor, but it might be a little more clear to flip the direction of the conditional:

```suggestion
        self._crs = CRS.from_user_input(crs) if crs else None
```

Reply:
Yeah, agreed, I mostly copied it from the other occurences (we can change those later if we want)